samplefuncs.py

```diff
@@ -71,6 +71,47 @@ def TopicTargetList(topic=None, data=None, srv=None):
 
     return targets
 
+def PushoverAddImageUrl(data=None, srv=None):
```

Comment:
I think this should be removed actually. Pretty specific to your use-case and doesn't really fit the bill as a simple sample function. I think the examples you added in README.md are sufficient.